__init__.py

```diff
@@ -1806,11 +1808,11 @@ def __to_time_period(self, when):
         if period is None:
             self.log.error("Unable to parse time as a period of day")
         return period
-    
+
     # Suggestion TODO: Add a parameter to extract_datetime to add a default Timezone
     def __extract_datetime(self, text, anchorDate=None, lang=None, default_time=None):
         # Change timezone returned by extract_datetime from Local to UTC
-        when, text = extract_datetime(text, anchorDate, lang, default_time)
+        when, text = extract_datetime(text, anchorDate, lang, default_time) or (None, None)
         return self.__to_UTC(when), text
```

Comment:
Will  `self.__to_UTC(when)` handle the None? looks like it would fail or am I missing that change?

Reply:
Yep good call, revised.